javascript/Color.Vision.Daltonize.js

```diff
@@ -38,8 +38,8 @@ Color.Vision.Daltonize = function(image, options) {
 		amount = typeof options.amount == "number" ? options.amount : 1.0,
 		canvas = document.createElement("canvas"),
 		ctx = canvas.getContext("2d");
-	canvas.width = image.width;
-	canvas.height = image.height;
+	canvas.width = image.naturalWidth;
```

Comment:
I think these properties aren't always available, so we should probably check to see if it's 0 / undefined before using it?

Reply:
I've added that change. For future reference, is there a list of browsers documented anywhere that webDiplomacy aims to support?